src/main/java/net/dries007/tfc/objects/blocks/wood/BlockSupport.java

```diff
@@ -74,29 +80,27 @@ public static ItemStack get(Tree wood, int amount)
      */
     public static boolean isBeingSupported(World worldIn, BlockPos pos)
     {
-        if (!worldIn.isAreaLoaded(pos.add(-32, -32, -32), pos.add(32, 32, 32)))
-        {
-            return true; //If world isn't loaded...
+        if (!worldIn.isAreaLoaded(pos.add(-32, -32, -32), pos.add(32, 32, 32))) {
+            return true; // If world isn't loaded...
```

Comment:
```suggestion
        if (!worldIn.isAreaLoaded(pos.add(-32, -32, -32), pos.add(32, 32, 32)))
        {
            return true; // If world isn't loaded...
```
Formatting. If you're using IDEA please import + use `CodeStyle.xml` found in the root of the repository and try and adhere to our code style.